app/players/player.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> str:
+        pass
+
+    @abstractmethod
+    def player_info(self) -> int:
```

Comment:
The `player_info` method should return a `str` instead of an `int`, as seen in the implementations of the subclasses. Ensure the return type matches across all implementations.